Dockerfile

```diff
@@ -30,17 +31,22 @@ RUN ./install_bioc.r \
 	corrplot \
 	cowplot \
 	cutpointr \
+	ggalluvial \
 	ggpubr \
 	ggthemes \
 	ggstatsplot \
 	ggfortify \
 	ggrepel \
 	GenomicFeatures \
+	GSVA \
 	msigdbr \
+	readr \
 	reshape2 \
 	R.utils \
 	survival \
-	survminer
+	survminer \
+	tibble \
+	tidyverse
```

Comment:
```suggestion
	survminer
```

Reply:
Good point. Thanks for the suggestion. I revised the Dockerfile. Thanks.